jdbc/build.gradle

```diff
@@ -33,11 +33,29 @@ dependencies {
     compile('org.postgresql:postgresql:9.4.+')
     compile('com.zaxxer:HikariCP:2.4.3')
 
-    compile('net.rakugakibox.springbootext:spring-boot-ext-logback-access:1.0')
+    compile('net.rakugakibox.springbootext:spring-boot-ext-logback-access:1.+')
+
+    testCompile('com.netflix.feign:feign-core:8.16.+')
```

Comment:
I'm not very fan on applying logic to get minor versions automatically, because the build isn't immutable at all. If feign updates to a new minor version, I will get the last minor and maybe this minor version has a very rare case and generates a problem in my project. Finding such cases it will be a PITA.

Ofcourse a minor version shouldn't break my code, but who knows, maybe I'm coupled with a "feature" that was a bug in the feign point of view.